github.go

```diff
@@ -91,6 +93,7 @@ func NewGitHubClient(source Source) (*GitHubClient, error) {
 			v4URL = source.GitHubAPIURL + "graphql"
 		}
 		clientV4 = githubv4.NewEnterpriseClient(v4URL, httpClient)
+		isEnterprise = true
 	}
 
 	if source.GitHubV4APIURL != "" {
```

Comment:
should `isEnterprise` be set here as well? Since here you are also creating a v4 client

Reply:
Done